net-mail/poppassd_ceti/poppassd_ceti-1.8.5-r3.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs pam portability
+
+MY_PN="poppassd"
+MY_P="${MY_PN}-${PV}"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Password change daemon with PAM support"
+HOMEPAGE="http://echelon.pl/pubs/poppassd.html"
```

Comment:
This didnt work for me. Is it just temporarily down, or gone for good? Any alternatives?

Reply:
I think the link is completely dead.

The only reason I'm updating this ebuild is because of a new policy to apparently either update or remove all old EAPI=0 ebuilds, and I don't want it removed.

Most of the top google results are for the Ubuntu / lauchpad version of this package, or for independent implementations of the same conceptual service.  Nothing looks like a project web page for this package specifically...

I've been using previous -rX version(s) (maybe just 1 version?) on a server for nearly 15 years without worrying about where it came from; it was probably picking it up from gentoo's mirrors.

What's the appropriate thing to do for a URL in this circumstance?

Reply:
Seems to me that the project moved to Github years ago: https://github.com/kravietz/poppassd-ceti  (And it looks like it's had a release or two since then as well.)

It also adopted autotools since then, which may make your `src_compile()` more standardized.

Reply:
Thanks.  I didn't search for the correct keywords, and didn't find that (underscore instead of dash, didn't try author name, etc).

I'll look into replacing my pull request with the latest release instead...

Reply:
Yes please :)

We'd also prefer if you became the maintainer for it, so it won't be left hanging anymore.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml
just edit the metadata.xml file in a separate commit or so.